index.html

```diff
@@ -1180,25 +1185,25 @@ <h2>Status</h2>
         </pre>
 
         <p>
-Defining the data model, formats, and protocols for status schemes are out
-of scope for this specification. A status scheme registry [[VC-STATUS-REGISTRY]]
-exists for implementers that would like to implement credential status checking.
+Defining the data model, formats, and protocols for status schemes are out of 
+scope for this specification. A status scheme registry [[VC-STATUS-REGISTRY]]
+exists for implementers who want to implement credential status checking.
         </p>
       </section>
 
       <section>
-        <h2>Presentations</h2>
+        <h3>Presentations</h3>
 
         <p>
-Credentials MAY be used to construct a <a>verifiable presentation</a>. A
-verifiable presentation may be of a verifiable credential itself, or
-of data derived from verifiable credentials in a format that is
-cryptographically verifiable.
+<a>Credentials</a> MAY be used to construct a <a>verifiable presentation</a>. 
+A <a>verifiable presentation</a> can be of a <a>verifiable credential</a> 
```

Comment:
We have the same two conformance issues here. 
1. Presentations MAY be used to combine VCs.
2. If Presentations are used, then they MUST contain VCs. 
i.e. the use of presentations is optional, but the format of presentations is standardised.